notes/coredns-1.8.4.md

```diff
@@ -3,13 +3,17 @@ title = "CoreDNS-1.8.4 Release"
 description = "CoreDNS-1.8.4 Release Notes."
 tags = ["Release", "1.8.4", "Notes"]
 release = "1.8.4"
-date = 2021-05-14T07:00:00+00:00
+date = 2021-05-18T07:00:00+00:00
 author = "coredns"
 +++
 
 The CoreDNS team has released
 [CoreDNS-1.8.4](https://github.com/coredns/coredns/releases/tag/v1.8.4). This release includes a
-bunch of bugfixes and a few enhancements, and a new (small) plugin called *minimal*.
+bunch of bugfixes and a few enhancements mostly in the *dnssec* and *kubernetes* plugins, and a new
+(small) plugin called *minimal*.
+
+It also include a fix when using the "reverse zone syntax", i.e. 10.0.0.0/15, now return the proper
```

Comment:
Should highlight required changes to existing external plugins introduce with the plugin level reverse zone changes.